tests/ctst/common/hooks.ts

```diff
@@ -17,11 +21,88 @@ const noParallelRun = atMostOnePicklePerTag(['@AfterAll']);
 setParallelCanAssign(noParallelRun);
 
 Before(async function (this: Zenko) {
+    this.resetSaved();
+    Identity.resetIdentity();
     await Zenko.init(this.parameters);
 });
 
-After(function (this: Zenko) {
-    this.resetSaved();
+/**
+ * The objective of this hook is to prepare all the buckets and accounts
+ * we use during quota checks, so that we avoid running the job multiple
+ * times, which affects the performance of the tests.
+ * The steps are: create an account, then create a simple bucket
+ */
+Before({ tags: '@Quotas', timeout: 1200000 }, async function ({ gherkinDocument, pickle }) {
```

Comment:
this before hook is specific to `Quotas` : can we define it in `quotas.ts` instead?

Reply:
Not really, the hooks in cucumber are evaluated in the same order they are defined, and this hook must be placed here ... Having it in a separate file will cause issues
There is no "ordering" feature yet in cucumber

Reply:
The reason we need to have it here, is that the main `Before` hook is defined just above, and is required before anything starts, as it is the one that init the Zenko world class, and reset the identity from a previous test.

Reply:
:-(

Could we keep the `Before()` here, but move the body (i.e. the async function) in the `quota.ts` ?
it would address the locality aspect, but maybe introduce a (blocking?) dependency loop?

Reply:
I will try that, I think it should work...

Reply:
I made the change, it passes the local build, so we should be good, all branches are updated, and I also moved the teardown hook